types/node/stream.d.ts

```diff
@@ -212,12 +212,12 @@ declare module "stream" {
             _writev?(chunks: Array<{ chunk: any, encoding: string }>, callback: (error?: Error | null) => void): void;
             _destroy(error: Error | null, callback: (error: Error | null) => void): void;
             _final(callback: (error?: Error | null) => void): void;
-            write(chunk: any, cb?: (error: Error | null | undefined) => void): boolean;
             write(chunk: any, encoding?: string, cb?: (error: Error | null | undefined) => void): boolean;
+            write(chunk: any, cb?: (error: Error | null | undefined) => void): boolean;
             setDefaultEncoding(encoding: string): this;
             end(cb?: () => void): void;
             end(chunk: any, cb?: () => void): void;
-            end(chunk: any, encoding?: string, cb?: () => void): void;
+            end(chunk: string, encoding?: string, cb?: () => void): void;
```

Comment:
I think if `write(chunk: any, encoding?: string,...)` exists the same should be allowed for `end()`